LICENSE.txt

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2000-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+   Copyright 2016-2024 JetBrains s.r.o. and Kotlin Programming Language contributors.
```

Comment:
I thought it was okay to edit the part of `LICENSE` after "END OF TERMS AND CONDITIONS", but since we do have `NOTICE`, I guess there's no reason to do so.